test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/ProjectDirectory.cs

```diff
@@ -18,9 +18,9 @@ internal class ProjectDirectory : IDisposable
         public bool PreserveWorkingDirectory { get; set; }
 #endif
 
-        public static ProjectDirectory Create(string projectName, string[] additionalProjects)
+        public static ProjectDirectory Create(string projectName, string baseDirectory, string[] additionalProjects)
         {
-            var destinationPath = Path.Combine(Path.GetTempPath(), "Razor", Path.GetRandomFileName());
+            var destinationPath = Path.Combine(Path.GetTempPath(), "Razor", baseDirectory, Path.GetRandomFileName());
```

Comment:
This does miss the scenario of a project file with spaces. We can do some work to cover that scenario in a test since that's also an interesting scenario to cover.

Reply:
Can we just rename `SimpleMvc` to `Simple Mvc`? I don't see any reason why not.